test/tools/unified-spec-runner/match.ts

```diff
@@ -735,7 +735,9 @@ export function expectErrorCheck(
   }
 
   if (expected.errorContains != null) {
-    expect(error.message, expectMessage).to.include(expected.errorContains);
+    expect(error.message.toLowerCase(), expectMessage.toLowerCase()).to.include(
+      expected.errorContains.toLowerCase()
```

Comment:
Unified test runner spec states this check should be case insensitive, so I just fixed it here as a few tests were failing because of it.